code/game/objects/structures/ladders.dm

```diff
@@ -187,6 +187,25 @@
 	if(AM)
 		user.start_pulling(AM)
 
+/obj/structure/ladder/densityBypass()
+	return TRUE
```

Comment:
This is not a good solution.
Do a (timed) attackby() instead, like Bay servers do, else this has the risk of sucking in unintended throws.

Reply:
Does attackby() will catcks stuff being throwing ? cause, the ladder has no density also is anchored